lib/Raven/Client.php

```diff
@@ -276,6 +276,17 @@ public function setEnvironment($value)
         return $this;
     }
 
+    /**
+     * Note: Prior to PHP 5.6, a stream opened with php://input can
```

Comment:
Since you're targeting the master branch, PHP < 5.6 is allowed, so you should read that only if PHP is >= 5.6; otherwise we would mess with the app.

Reply:
It won't really 'mess up the app'. What will happen is the data will be null. I can adjust the comment to explain that. This doesn't introduce a regression issue because the previous implementation and the current one still return the same broken results for php from the dark ages :)

I can add the condition in. I felt that this is really an edge case and there is no clean way to handle support for PHP < 5.6 with this feature.

Reply:
The one thing that is a little strange is the unit tests will succeed in < 5.6 but the reality is the feature doesn't work in that version. Not sure if there is a way to resolve that. Just a thought.

(I added the conditional https://github.com/getsentry/sentry-php/pull/546/files#diff-11a1d8314f22caf150ad02a00b294b58R287 )

Reply:
The data will be empty on < 7.0 but it can also be we fetched the data before the user could (if the use the library for logging for example) causing it to possibly break the user's application.

And yes this will not go well for PHP 5.6 but we should try and solve that differently if we must. But this could affect the implementors application and that is something we really should avoid at all cost.